debian/cortx-motr.install

```diff
@@ -0,0 +1,116 @@
+usr/lib/x86_64-linux-gnu/libmotr-helpers.so.0
+usr/lib/x86_64-linux-gnu/libmotr.so.2
+usr/lib/x86_64-linux-gnu/libmotr-helpers.so.0.0.0
+usr/lib/x86_64-linux-gnu/libm0isc.so.1
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-service.functions
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-trace
+usr/lib/x86_64-linux-gnu/cortx-motr/motr
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-cleanup
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-start
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-mkfs
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-kernel
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-free-space-monitor
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-client
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-server
+usr/lib/x86_64-linux-gnu/cortx-motr/motr-dixinit
+usr/lib/x86_64-linux-gnu/libmotr.so
+usr/lib/udev/rules.d/99-motr.rules
+usr/sbin/m0gendisks
+usr/sbin/m0beck
+usr/sbin/m0protocol
+usr/sbin/m0ham
+usr/sbin/m0console
+usr/sbin/m0genfacts
+usr/sbin/m0kv
+usr/sbin/m0composite
+usr/sbin/m0betool
+usr/sbin/m0dixinit
+usr/sbin/m0kemc
+usr/sbin/m0iscorrupt
+usr/sbin/m0mkfs
+usr/sbin/m0crate
+usr/sbin/m0reportbug
+usr/sbin/m0repair
+usr/sbin/m0run
+usr/sbin/m0provision
+usr/sbin/m0singlenode
+usr/sbin/m0tracepurge
+usr/sbin/m0addb2dump
+usr/sbin/m0layout
+usr/sbin/m0poolmach
+usr/sbin/m0setup
+usr/share/motr/protocol-be.md5
+usr/share/motr/protocol-rpc.md5
+usr/share/motr/protocol-be.txt
+usr/share/motr/protocol-conf.txt
+usr/share/motr/protocol.md5
+usr/share/motr/protocol.txt
+usr/share/motr/protocol-conf.md5
+usr/share/motr/protocol-rpc.txt
+usr/share/man/man1/m0d.1
+usr/share/man/man1/m0traced.1
+usr/share/man/man1/m0tracedump.1
+usr/share/man/man1/m0trace.1
+usr/share/man/man7/motr.7
+usr/bin/m0iscdemo
+usr/bin/m0tracedump
+usr/bin/m0kdump2trace
+usr/bin/m0d
+usr/bin/m0spiel
+usr/bin/m0cp_mt
+usr/bin/m0hagen
+usr/bin/m0unlink
+usr/bin/m0trunc
+usr/bin/m0trace
+usr/bin/m0cat
+usr/bin/m0cp
+usr/bin/m0hsm
+usr/bin/m0mt
+usr/bin/m0confgen
+usr/bin/m0client
+usr/bin/m0touch
+usr/bin/m0_filesystem_stats
+usr/bin/m0traced
+usr/bin/m0gentestds
+usr/bin/m0iscreg
+opt/seagate/cortx/motr/libexec/m0trace_logrotate.sh
+opt/seagate/cortx/motr/libexec/m0_ha_sim
+opt/seagate/cortx/motr/libexec/motr_data_recovery.sh
+opt/seagate/cortx/motr/libexec/motr_cfg.sh
+opt/seagate/cortx/motr/libexec/m0addb_logrotate.sh
+opt/seagate/cortx/motr/libexec/m0reportbug-bundler
+opt/seagate/cortx/motr/common/cortx_error_codes.sh
+opt/seagate/cortx/motr/common/m0_sns_utils_common.sh
+opt/seagate/cortx/motr/common/cortx_util_funcs.sh
+opt/seagate/cortx/motr/common/core-traces
+etc/systemd/system/motr-mkfs.service
+etc/systemd/system/motr-client.service
+etc/systemd/system/motr-server-confd.service
+etc/systemd/system/motr-server-ha.service
+etc/systemd/system/motr.service
+etc/systemd/system/motr-singlenode.service
+etc/systemd/system/motr-server@.service
+etc/security/limits.d/90-motr.conf
+etc/cron.hourly/m0trace_logrotate.sh
+etc/cron.hourly/m0addb_logrotate.sh
+etc/sysconfig/motr-free-space-monitor
```

Comment:
etc/sysconfig/motr too is needs to be added here.

Reply:
Added.